### PR TITLE
jormungandr: Test on departures and stop_schedules added

### PR DESCRIPTION
**Pull request by kadhikari**
For details: https://jira.kisio.org/browse/NAVP-1194

### CODE REVIEW

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 1]
```diff
@@ -1557,6 +1558,20 @@ def test_add_new_trip(self):
...
 
+        # Check that no departure exist on stop_point stop_point:stopC
+        departure_query = "stop_points/stop_point:stopC/departures?_current_datetime=20120614T080000"
+        departures = self.query_region(departure_query)
+        assert len(departures['departures']) == 0
+
+        # Check that no stop_schedule exist on line:additional_service and stop_point stop_point:stopC
+        ss_query = (
+            "stop_points/stop_point:stopC/lines/line:additional_service/"
+            "stop_schedules?_current_datetime=20120614T080000&data_freshness=realtime"
```
Comment (pbougue) [inline source/jormungandr/tests/kirin_realtime_tests.py]: I vote to also add a check using `base_schedule` to check that we don't contaminate it (departures and/or stop_schedules).

**Review by pbougue — COMMENTED** [review]
Nice work!
If you don't think it's worthy to add a check on base_schedule, just ask a third party to decide :wink:

**Comment by pbougue** [comment]
Jenkins, test this please.

**Comment by benoit-bst** [comment]
Jenkins, test this please !

File: source/jormungandr/tests/kirin_realtime_tests.py (hunk 1 above)
Comment (benoit-bst) [reply source/jormungandr/tests/kirin_realtime_tests.py]: This is not in the scope of the _JIRA ticket_, so normally we should leave as is, and create an other PR if it necessary, according to the rules that we have mentionned in Meeting (_Talk to the team -> create new ticket if necessary -> create new PR_).
But I think, it's an oblivion. We should test this _entry point_ in the ticket. So, I am agree with you @pbougue.

**Review by benoit-bst — APPROVED** [review]

File: source/jormungandr/tests/kirin_realtime_tests.py (hunk 1 above)
Comment (kadhikari) [reply source/jormungandr/tests/kirin_realtime_tests.py]: I added some more verifications as asked

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 2]
```diff
@@ -1656,19 +1667,49 @@ def test_add_new_trip(self):
         assert len(response['networks']) == 1
         assert response['networks'][0]['name'] == 'additional service'
 
+        # Check that no departure exist on stop_point stop_point:stopC for base_schedule
```
Comment (pbougue) [inline source/jormungandr/tests/kirin_realtime_tests.py:1670]: minor typo: exist`s`

**Review by pbougue — COMMENTED** [review]
:+1:

**Review by pbougue — APPROVED** [review]

**Pull request merged**